go/app/main.go

```diff
@@ -45,19 +138,77 @@ func getImg(c echo.Context) error {
 		return c.JSON(http.StatusBadRequest, res)
 	}
 	if _, err := os.Stat(imgPath); err != nil {
-		c.Logger().Debugf("Image not found: %s", imgPath)
+		c.Logger().Errorf("Image not found: %s", imgPath)
 		imgPath = path.Join(ImgDir, "default.jpg")
 	}
 	return c.File(imgPath)
 }
 
 func main() {
+	var err error
+	db, err = sql.Open("sqlite3", "../../db/mercari.sqlite3")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer db.Close()
+
+	err = db.Ping()
+	if err != nil {
+		panic(err)
+	}
+
+	//print items table
+	rows, err := db.Query("SELECT * FROM items")
+	if err != nil {
+		panic(err)
+	}
+	defer db.Close()
+
+	var id int
+	var name string
+	var categoryId int
+	var image string
+	fmt.Println("Items:")
```

Comment:
Debug用でこれを残したいようでしたら、
```suggestion
	c.Logger().Debugf("Items:")
```
を使うのがいいですね。
printlnは開発中は使ってもいいですが、PRにするときはない方がベターです◎

Reply:
ありがとうございます。勉強になります。